examples/envs/lung/balloon_lung_pid.ipynb

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
Mind just dropping the `_pid` from the lung notebooks? Otherwise, looks good!

Reply:
In the upcoming notebooks, I have notebooks such as LDS_GPC, LDS_Hinf, LDS_adapative, etc. which is why I choose to name them in the form env_method to begin with. Isn't it useful for people looking for notebooks to see both env and method in the name?

Reply:
Ah, I see, understood; makes sense!